PhysicsTools/PatUtils/src/MiniIsolation.cc

```diff
@@ -55,19 +55,21 @@ PFIsolation getMiniPFIsolation(const pat::PackedCandidateCollection *pfcands,
 
 }
 
-  float muonRelMiniIsoPUCorrected(const PFIsolation& iso,
-				  const math::XYZTLorentzVector& p4,
-				  float dr,
-				  float rho)
+  double muonRelMiniIsoPUCorrected(const PFIsolation& iso,
+				   const math::XYZTLorentzVector& p4,
+				   double dr,
+				   double rho,
+                                   std::vector<double> &EA)
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.
Also, keep the vector const.
==> ```const std::vector<double> &ea```